radixdlt-engine/src/main/java/com/radixdlt/constraintmachine/REParsedTxn.java

```diff
@@ -100,4 +100,9 @@ public boolean equals(Object o) {
 		return Objects.equals(this.txn, other.txn)
 			&& Objects.equals(this.actions, other.actions);
 	}
+
+	@Override
+	public String toString() {
+		return "REParsedTxn(" + txn + ", " + actions + ')';
```

Comment:
Lets use the format we've been using throughout the code:
```
String.format("%s{%s=%s}", this.getClass().getSimpleName())...
```

Reply:
Updated